core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1482,9 +1481,9 @@ public synchronized void onBlocksDownloaded(Peer peer, Block block, @Nullable Fi
 
         @Override
         public synchronized void run() {
-            if (blocksInLastSecond > 1) {
```

Comment:
Single blocks are also "download". If your goal is excluding blocks received by "inv" rather than "getdata" then the way to go is clearly use *that* info for the conditional, rather than introducing a subtle bug into the logging. As I said, I was missing the message and invested quite some time into debugging.